ui/shared/components/panel/family.jsx

```diff
@@ -34,7 +31,6 @@ import {
   FAMILY_FIELD_RENDER_LOOKUP,
   FAMILY_FIELD_OMIM_NUMBER,
```

Comment:
everything moved to ` ui/pages/Project/components/FamilyPage.jsx` which is the only place it is ever used